src/Register.php

```diff
@@ -0,0 +1,43 @@
+<?php namespace Lean\Elements;
+
+/**
+ * Class to provide predefined ACF elements.
+ */
+class Register
+{
+	/**
+	 * Registered elements.
+	 *
+	 * @var array
+	 */
+	private static $_elements = [];
+
+	/**
+	 * Init. Takes the location and the elements to register in that location.
+	 *
+	 * @param array $location Array suitable for ACF location parameter.
+	 * @param arrat $elements Array of element names to show in this location.
+	 */
+	public static function add( $element, $location ) {
```

Comment:
What about social, contact, site logo - can we have a default location for these? E.g. Options > Your Organization page.